main_test.go

```diff
@@ -39,5 +40,13 @@ func commonEnv(env *testscript.Env) error {
 	env.Setenv("HOME", "$WORK/home")
 	env.Setenv("VERSION", static.Version)
 	env.Setenv("COMMIT_SHA", static.Sha)
+	tb, ok := env.T().(testing.TB)
+	if !ok {
+		env.T().Skip("Unable to make a database")
+		return nil
+	}
+	cfg, _ := heavyweight.FullTestDBEmptyV2(tb, nil)
```

Comment:
I think I can simplify this a lot, since we aren't using the fixtures or even migrations.. We should be able to avoid moving `package heavyweight`, and instead move/mimic only the randomized DB logic. :thinking:

Reply:
Also:
- [ ] make this conditional (test name? dir?)